### ISSUE TITLE
[BUG] TikTokLiveManager.AvailableGifts is null - conversion failed

### ISSUE THREAD

**Issue by IgorAherne** [frankvHoof93/TikTokLiveSharp#45]
Hello, `TikTokLiveManager.Instance.AvaialbleGifts` always return null, even if `TikTokLiveManager.Exists` is true and it's connected and working otherwise.

```
TikTokLiveManager{
        public IReadOnlyDictionary<int, TikTokGift> AvailableGifts => (IReadOnlyDictionary<int, TikTokGift>)client?.AvailableGifts;
}
```

When checking with breakpoints, it's saying null because the specified cast is not valid

**Comment by frankvHoof93** [comment]
1. Are you sure that 'DowbloadGiftInfo' is set to true on the TTLManager?
2. Could you put a breakpoint in TTBaseClient to check what the JSON-Response is that it's receiving?

**Comment by IgorAherne** [comment]
@frankvHoof93 Can't do it, it won't let me place it because it's from metadata:
![image](https://github.com/frankvHoof93/TikTokLiveSharp/assets/17007129/8c67cfc5-4d53-434f-95c5-75b203c8b248)

**Comment by IgorAherne** [comment]
However, here is more info from the watch window, at the breakpoint above

I'm not sure how to find it man :D
_Are you sure that 'DowbloadGiftInfo' is set to true on the TTLManager_

![image](https://github.com/frankvHoof93/TikTokLiveSharp/assets/17007129/77a7f352-b4fd-400e-8ecd-e6923f7f119b)

**Comment by IgorAherne** [comment]
Could it be failing the cast because it confuses it for the `TikTokLiveSharp.Models.HTTP.TikTokGift` instead of the `TikTokLiveSharp.Events.MessageData.Objects.TikTokGift` ?

**Comment by rsputra24** [comment]
yes it was because of ambiguity between those two, import it like this 
using TikTokLiveSharpGift = TikTokLiveSharp.Models.HTTP.TikTokGift;
then cast it to TikTokLiveSharpGift

**Comment by frankvHoof93** [comment]
This should be resolved in V1.0.0.
There is now also an implicit operator that will cast from the HTTP-TikTokGift to the Event-TikTokGift if needed.
Please let me know if the issue persists in V1.0.0 by opening a new Github-Issue
